sqlglot/dialects/trino.py

```diff
@@ -63,6 +68,7 @@ class Generator(Presto.Generator):
             e: f"REDUCE({self.sql(e, 'this')}, 0, (acc, x) -> acc + x, acc -> acc)",
             exp.ArrayUniqueAgg: lambda self, e: f"ARRAY_AGG(DISTINCT {self.sql(e, 'this')})",
             exp.Merge: merge_without_target_sql,
+            exp.StrPosition: lambda self, e: strposition_sql(self, e, supports_occurrence=True),
```

Comment:
This should be inherited from Presto, so changes in this file can be reverted altogether.